quarkusQS/pom.xml

```diff
@@ -0,0 +1,191 @@
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.wildfly.extras.sunstone</groupId>
+  <artifactId>quarkusHelloWorld</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <properties>
+    <compiler-plugin.version>3.10.1</compiler-plugin.version>
+    <maven.compiler.release>11</maven.compiler.release>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+    <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+    <quarkus.platform.version>2.16.6.Final</quarkus.platform.version>
+    <surefire-plugin.version>3.0.0-M7</surefire-plugin.version>
+      <maven.compiler.target>11</maven.compiler.target>
+      <failOnMissingWebXml>false</failOnMissingWebXml>
+      <maven.compiler.source>11</maven.compiler.source>
+      <final.name>demo</final.name>
+      <version.quarkus>1.7.6.Final</version.quarkus>
+      <azure.subscriptionId>wwwwwwww-wwww-wwww-wwww-wwwwwwwwwwww</azure.subscriptionId>
+      <azure.clientId>xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx</azure.clientId>
+      <azure.tenantId>yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy</azure.tenantId>
+      <azure.password>zzzzzzzz-zzzz-zzzz-zzzz-zzzzzzzzzzzz</azure.password>
+      <!-- to guarantee test instance Name uniqueness, supports: https://github.com/wildfly-extras/sunstone/issues/154 -->
+      <timestamp>${maven.build.timestamp}</timestamp>
+      <maven.build.timestamp.format>yyyyMMddHHmmss</maven.build.timestamp.format>
+      <azure.group>SunstoneQuarkusWebApp-${timestamp}</azure.group>
+      <azure.appName>quarkus-helloWorld-${timestamp}</azure.appName>
+  </properties>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>${quarkus.platform.group-id}</groupId>
+        <artifactId>${quarkus.platform.artifact-id}</artifactId>
+        <version>${quarkus.platform.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-arc</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-reactive</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-junit5</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <!-- https://mvnrepository.com/artifact/org.wildfly.extras.sunstone/sunstone-azure -->
+    <dependency>
+      <groupId>org.wildfly.extras.sunstone</groupId>
+      <artifactId>sunstone-azure</artifactId>
+      <version>2.0.2-SNAPSHOT</version>
```

Comment:
You should define parent and omit the version using maven dependenccy defined in parent's dependency management.